src/amp/server/render.tsx

```diff
@@ -65,8 +65,11 @@ export const render = ({ body }: express.Request, res: express.Response) => {
 
         res.status(200).send(resp);
     } catch (e) {
-        // a validation error
-        if (e instanceof TypeError) {
+        if (e.message.startsWith('404')) {
```

Comment:
Could we avoid this by creating a custom error type or using the `name` property? The latter would still be a string comparison, but at least it's more obvious that it shouldn't be changed without checking elsewhere.

Reply:
Agree with Jon, but we can card it for later :)